max_sub_array/max_sub_array.py

```diff
@@ -0,0 +1,16 @@
+def max_sub_array_sum(a, size):
+
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpemchpeeu/max_sub_array/max_sub_array.py
+++ b/tmp/tmpemchpeeu/max_sub_array/max_sub_array.py
@@ -1,5 +1,4 @@
 def max_sub_array_sum(a, size):
-
 
     max_so_far = -9999999 - 1
     max_ending_here = 0
```